src/app/routes/index.test.jsx

```diff
@@ -397,6 +397,8 @@ it('should route to and render a story page', async () => {
     storyPageRecommendationsData,
   );
 
+  jest.setTimeout(30000);
```

Comment:
Strange side effect from upgrading, is this unit test takes longer to resolve promises, hence the addition of increasing the timeout limit which was 5 seconds, not sure why this is the case

Reply:
Was looking into this too 😅 https://github.com/bbc/simorgh/pull/10336 

This has been a bit of a flakey test since the Theming was introduced. Not entirely sure what the root cause is.